library/src/howMuch.test.ts

```diff
@@ -0,0 +1,7 @@
+import { howMuch } from "./howMuch";
+
+describe('howMuch()', () => {
+  it('should return some results when searching for iPad', () => {
+    expect(howMuch('iphone').numResults).toBeGreaterThan(0)
```

Comment:
It's one or the other haha

```suggestion
  it('should return some results when searching for iPhone', () => {
    expect(howMuch('iphone').numResults).toBeGreaterThan(0)
```

```suggestion
  it('should return some results when searching for iPad', () => {
    expect(howMuch('iPad').numResults).toBeGreaterThan(0)
```